From 72494cf38c0c21299be893a01d8cd076e080fa0b Mon Sep 17 00:00:00 2001
From: Tobias Dubois <tobias.dubois@sony.com>
Date: Mon, 11 Jun 2018 16:25:55 +0200
Subject: [PATCH 06/32] Fix overlapping FloatingViewType IntDef

TYPE_DISCOVERY_BOUNCE and TYPE_QUICKSTEP_PREVIEW have the same value.
Make sure each flag has a unique value.

Test: manual
Change-Id: I0fec3f27ffc31bad80d3af7b3baa3631b61b2567
---
 src/com/android/launcher3/AbstractFloatingView.java | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/com/android/launcher3/AbstractFloatingView.java b/src/com/android/launcher3/AbstractFloatingView.java
index daf20323d..b614ec9d6 100644
--- a/src/com/android/launcher3/AbstractFloatingView.java
+++ b/src/com/android/launcher3/AbstractFloatingView.java
@@ -68,9 +68,9 @@ public abstract class AbstractFloatingView extends LinearLayout implements Touch
     public static final int TYPE_DISCOVERY_BOUNCE = 1 << 6;
 
     // Popups related to quickstep UI
-    public static final int TYPE_QUICKSTEP_PREVIEW = 1 << 6;
-    public static final int TYPE_TASK_MENU = 1 << 7;
-    public static final int TYPE_OPTIONS_POPUP = 1 << 8;
+    public static final int TYPE_QUICKSTEP_PREVIEW = 1 << 7;
+    public static final int TYPE_TASK_MENU = 1 << 8;
+    public static final int TYPE_OPTIONS_POPUP = 1 << 9;
 
     public static final int TYPE_ALL = TYPE_FOLDER | TYPE_ACTION_POPUP
             | TYPE_WIDGETS_BOTTOM_SHEET | TYPE_WIDGET_RESIZE_FRAME | TYPE_WIDGETS_FULL_SHEET
-- 
2.17.1

